src/sonic-frr/Makefile

```diff
@@ -0,0 +1,24 @@
+.ONESHELL:
+SHELL = /bin/bash
```

Comment:
In my current proposal, 'sonic-frr' is the wrapper folder containing the SONiC specific information required to build frr. Whereas 'frr' subfolder contains the cloned frr repo. Please let me know if you have any other suggestion for the naming convention.